docs/structured-streaming-kafka-integration.md

```diff
@@ -512,6 +526,17 @@ The following configurations are optional:
 </tr>
 </table>
 
+### Details on timestamp offset options
+
+The returned offset for each partition is the earliest offset whose timestamp is greater than or equal to the given timestamp in the corresponding partition.
+The behavior varies across options if the matched offset doesn't exist - check the description of each option.
+
+Spark simply passes the timestamp information to <code>KafkaConsumer.offsetsForTimes</code>, and doesn't interpret or reason about the value.
+For more details on <code>KafkaConsumer.offsetsForTimes</code>, please refer <a href="https://kafka.apache.org/21/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#offsetsForTimes-java.util.Map-">javadoc</a> for details.
```

Comment:
Putting exact version information into the doc needs time to time attention.
`https://kafka.apache.org/21/...`
If we assume Kafka is not breaking API then we can put `latest` instead of `21`. Though not sure Kafka has such link.
BTW, why `21`, we're on `<kafka.version>2.8.0</kafka.version>` and feature requires minimum `0.10.1.0`?

Reply:
If I understand correctly, there's no notion of "latest" so we picked the version we used at that time. (Worth noting that the content was added when we added timestamp offset.)

I'm OK to either raising the version to the one we use in 3.2 or lowering the version to minimum.

Reply:
I've double checked and no "latest" found. I think from maintenance perspective it would be the best to lower the version to the minimum. Such case the API must remain the same and we don't have to touch it time to time. WDYT?

Reply:
That makes sense. Hopefully Kafka community looks to maintain all versions of doc so the risk of broken link is relatively low. I'll update the link to 0.10.1.x. Thanks!